src/program-editors2.bib

```diff
@@ -0,0 +1,56 @@
+@misc{cooke, title={A History of Accessibility at IBM}, url={https://www.afb.org/aw/5/2/14760}, journal={A History of Accessibility at IBM}, author={Cooke, Annemarie}}
+ 
+@misc{sanchez2005blind,
+  title={Blind learners programming through audio},
+  author={S{\'a}nchez, Jaime and Aguayo, Fernando},
+  booktitle={CHI'05 extended abstracts on Human factors in computing systems},
```

Comment:
* Is there a full paper about this same work or was it only an extended abstract?
* Capitalization

Reply:
There is some follow up work by the authors but it is not related to programming tasks specifically ("3D sound interactive environments for blind children problem solving skills", "3D sound interactive environments for problem solving"), so I chose to include this poster instead.